packages/app/pages/course/[cid]/queue/[qid].tsx

```diff
@@ -137,9 +137,8 @@ export default function Queue() {
     mutate(`/api/v1/queues/${qid}/questions`, newQuestions);
   };
 
-  // TODO: yup
-  const alertStudent = (question: Question) => {
-    // Send API request to trigger notification
+  const alertStudent = async (question: Question) => {
+    await API.questions.notify(question.id);
```

Comment:
is there a reason why we can't use the notification controller for this alert? we have the student's id from the question's creator property, we would just have to add a request body to store the message. i think this would also remove the need for a new endpoint in question controller

Reply:
nvm! just read the discussion on #33 and the issue #45 

makes sense to have a question notification endpoint to validate that the TA trying to alert is currently helping/in the queue